tests/cli/test_configure.py

```diff
@@ -12,30 +10,9 @@
 
 
 class TestConfigureCommand(TestCase):
+    @mock.patch("mbed_tools.cli.configure.generate_config")
     @mock.patch("mbed_tools.cli.configure.MbedProgram")
-    @mock.patch("mbed_tools.cli.configure.get_target_by_name")
-    @mock.patch("mbed_tools.cli.configure.generate_mbed_config_cmake_file")
-    @mock.patch("mbed_tools.cli.configure.assemble_config")
-    @mock.patch("mbed_tools.cli.configure.write_file")
-    def test_collaborators_called_with_corrrect_arguments(
-        self, write_file, assemble_config, gen_config_cmake, get_target_by_name, mbed_program
-    ):
-        CliRunner().invoke(configure, ["-m", "k64f", "-t", "GCC_ARM"])
+    def test_generate_config_called_with_correct_arguments(self, program, generate_config):
+        CliRunner().invoke(configure, ["-m", "k64f", "-t", "gcc_arm"])
```

Comment:
Does case matter here? Should we keep using GCC_ARM as before, and also start using K64F?

Reply:
Case does matter. The `targets` API needs the target name in uppercase. The toolchain is just passed through to mbed_config.cmake, I believe, and CMake also expects uppercase.